peps/pep-0735.rst

```diff
@@ -0,0 +1,490 @@
+  use these files by name.
+
+* ``requirementst.txt`` files are *not standardized*, but instead provide
+  options to ``pip``.
+
+Therefore, their use is not portable to any new
+installer or tool which wishes to process them without relying upon ``pip``.
+Additionally, ``requirements.txt`` files may be viewed as a heavyweight
+solution for very small dependency sets of only one or two items, and a terser
+declaration will be beneficial to projects with a number of small groups of
+dependencies.
+
+Regarding extras, the use of extras to define development dependencies is a
+widespread practice, but it has two major downsides.
+
+* An extra is defined as an additive group of dependencies for a package.
+  This means that it cannot be installed without installing all of the package
+  dependencies, and the project must be defined as a package.
+
+* Many developers view their extras as part of the public interface for their
+  package. Because these are published data, package developers often are
+  concerned about ensuring that their development extras are not confused with
+  user-facing extras. Therefore, their development needs are not appropriate to
+  publish in the manner of extras.
+
+Providing a standardized place to store dependency data which matches the
+typical use-cases of ``requirements.txt`` files will allow for better
+cross-compatibility between tools and will onboard beginning users into
+``pyproject.toml`` data more quickly (before they learn about the
+``build-system`` table, for example). It will also resolve the long-standing
+ambiguity and tension within the Python community about whether or not extras
+should be used to declare development dependencies.
+
+Supported Project Types
+-----------------------
+
+This PEP aims to serve the needs of two distinct project types:
+
+* libraries and other packages which want to define development dependencies
+
+* projects which do not build distributions which want a standardized way to
+  declare their dependency data
+
+The Python packaging toolchain is oriented towards the production of wheel and
+sdist files, containing project source for installation. However, not all
+projects are designed or intended for this mode of distribution. Primary
+examples include webapps (which often run from source), and data science
+projects (which may start as collections of scripts and only later evolve
+package structures). This PEP seeks to offer these projects a way to specify
+their package dependencies in a way which is not constrained by the
+requirements of distribution building -- for example, a data science project
+does not need to declare a version or authors in order for these data to be
+valid, nor does it need to define an installable package source tree.
+
+Simultaneously, this PEP should benefit projects which *are* intended to build
+distributions. Because these requirements data are explicitly defined to not be
```

Comment:
I think the right rephrase will be "Because Dependency Groups are..." -- the issue here is an unclear antecedent.

I wouldn't call it _very_ awkward, but it's definitely a case in which my pedantic streak (insisting that "data are" and "datum is") doesn't match some pretty common colloquial English.

Reply:
Reading this sentence I don't know what "these data" refers back to. Maybe this:
```suggestion
distributions. Because dependency groups, as specified in this PEP, are explicitly defined to not be
```

Reply:
I thought I fixed this line. I must have just missed it. :facepalm: 

Your edit is almost verbatim what I wanted to make this.